templates/ingress-apigee.yaml

```diff
@@ -17,23 +17,20 @@ metadata:
     nginx.ingress.kubernetes.io/proxy-connect-timeout: "{{.NginxIngressProxyConnectTimeout}}"
     nginx.ingress.kubernetes.io/proxy-send-timeout: "{{.NginxIngressProxySendTimeout}}"
     nginx.ingress.kubernetes.io/proxy-read-timeout: "{{.NginxIngressProxyReadTimeout}}"
-    {{- if .OverrideDefaultWhitelist}}
-    nginx.ingress.kubernetes.io/whitelist-source-range: "{{.NginxIngressWhitelist}}"
-    {{- end}}
     {{- if .SetsNginxIngressLoadBalanceAlgorithm }}
     nginx.ingress.kubernetes.io/load-balance: "{{.NginxIngressLoadBalanceAlgorithm}}"
     {{- end }} 
     estafette.io/cloudflare-dns: "true"
     estafette.io/cloudflare-proxy: "false"
-    estafette.io/cloudflare-hostnames: "{{.HostsJoined}}"
+    estafette.io/cloudflare-hostnames: "{{.ApigeeHostsJoined}}"
     nginx.ingress.kubernetes.io/auth-tls-pass-certificate-to-upstream: "true"
     nginx.ingress.kubernetes.io/auth-tls-secret: "{{.NginxAuthTLSSecret}}"
```

Comment:
So we going to use helm to publish common TLS secret to be used by all apigee ingresses for the client auth?

Maybe worth to add comments so we can understand better what we configure with these settings and why we use them :
https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/nginx-configuration/annotations.md#client-certificate-authentication

Reply:
I have already raised PR in travix-secrets yesterday to deploy ca cert. I'll check how I can document it where it will be available on Estafette.io.

Reply:
I think even leaving comment inside template would be enough - so next person working on it could better understand context, but this is not blocker just suggestion